index.js

```diff
@@ -2,21 +2,44 @@
  * Reverse Polish Notation Calculator
  */
 
-var arguments = process.argv.slice(2);
+let arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
-var operator = arguments[2];
+/* FIX #4 - Calculation is not possible with Strings */
+let operand1 = parseInt(arguments[0], 10);
+let operand2 = parseInt(arguments[1], 10);
```

Comment:
Why ParseInt(x,10)
No calc with decimal numbers

Reply:
@FearZee Please create a dedicated issue for that.